authn/pkce.index.js

```diff
@@ -362,9 +362,9 @@ function unauthorized(error, error_description, error_uri, callback) {
   </html>
   `;
 
-  page = page.replace(/%error%/g, error);
-  page = page.replace(/%error_description%/g, error_description);
-  page = page.replace(/%error_uri%/g, error_uri);
+  page = page.replace(/%error%/g, encodeURI(error).replace(/%20/g,' '));
```

Comment:
I can't see a token in the page for `%error%` so it this not necessary or should there be a replace token in the page?